main.ts

```diff
@@ -109,9 +133,85 @@ async function run(): Promise<void> {
     })) {
       ln(`/dev/${linkPath}`, `/proc/self/${target}`)
     }
+
+    if (msystem === 'CLANGARM64') {
+      // ARM64 dependencies aren't included yet in the Git for Windows SDK. Ask Pacman to install them.
+      core.startGroup(`Installing CLANGARM64 dependencies`)
+      await installArm64Dependencies(outputDirectory)
+      core.endGroup()
+    }
   } catch (error) {
     core.setFailed(error instanceof Error ? error.message : `${error}`)
   }
 }
 
-run()
+async function installArm64Dependencies(
+  outputDirectory: string
+): Promise<void> {
+  mkdirp(`${outputDirectory}/clangarm64`)
+  fs.appendFileSync(
+    `${outputDirectory}/etc/pacman.conf`,
+    `
+    [clangarm64]
+    Server = https://mirror.msys2.org/mingw/clangarm64/`
+  )
+
+  const child = spawn('pacman.exe', [
+    '-Sy',
+    '--noconfirm',
+    'base-devel',
+    'mingw-w64-clang-aarch64-openssl',
+    'mingw-w64-clang-aarch64-zlib',
+    'mingw-w64-clang-aarch64-curl',
+    'mingw-w64-clang-aarch64-expat',
+    'mingw-w64-clang-aarch64-libiconv',
+    'mingw-w64-clang-aarch64-toolchain',
+    'mingw-w64-clang-aarch64-pcre2',
+    'mingw-w64-clang-aarch64-libssp'
+  ])
+
+  child.stdout.on('data', data => {
+    core.info(data)
+  })
+
+  child.stderr.on('data', data => {
+    core.error(data)
+  })
+
+  return new Promise((resolve, reject) => {
+    child.on('error', error => reject(error))
+    child.on('close', status =>
+      status === 0
+        ? resolve()
+        : reject(new Error(`Process exited with status code ${status}`))
+    )
+  })
+}
+
+function cleanup(): void {
+  const {artifactName} = getArtifactMetadata(flavor, architectureToDownload)
+  const outputDirectory = core.getInput('path') || `C:/${artifactName}`
+
+  core.info(`Cleaning up ${outputDirectory}...`)
+  fs.rmSync(outputDirectory, {recursive: true, force: true})
+  core.info(`Finished cleaning up ${outputDirectory}.`)
+}
```

Comment:
This [takes ~1m34s in my testing](https://github.com/dennisameling/git/actions/runs/3073573453/jobs/4965793895). When using the native `rmdir /Q /S git-sdk-64-full`, it seems to take ~1 min. When using Git's `rm -rf git-sdk-64-full`, it takes only 20-30 seconds. Weird that it's faster than Windows' native tools.

Shall I use `rm -rf ...` if we can find it in `gitForWindowsUsrBinPath`, and otherwise fall back to `fs.rmSync`? That should speed up things quite a bit.

Reply:
With a corresponding code comment repeating what you said above, I am totally fine with shelling out to `rm -rf`.